src/platforms/common/configuration/sampling.mdx

```diff
@@ -154,7 +154,7 @@ There are multiple ways for a transaction to end up with a sampling decision.
 When there's the potential for more than one of these to come into play, the following precedence rules apply:
 
 1. If a sampling decision is passed to <PlatformIdentifier name="start-transaction" />, that decision will be used, overriding everything else.
-1. If <PlatformIdentifier name="traces-sampler" /> is defined, its decision will be used. It can choose to keep or ignore any parent sampling decision, or use the sampling context data to make its own decision or choose a sample rate for the transaction.
+1. If <PlatformIdentifier name="traces-sampler" /> is defined, its decision will be used. It can choose to keep or ignore any parent sampling decision, or use the sampling context data to make its own decision or choose a sample rate for the transaction. (We adive against overriding the parent sampling decision, because it will break distirbuted traces.)
```

Comment:
```suggestion
1. If <PlatformIdentifier name="traces-sampler" /> is defined, its decision will be used. It can choose to keep or ignore any parent sampling decision, or use the sampling context data to make its own decision, or choose a sample rate for the transaction. (We advise against overriding the parent sampling decision, because it will break distributed traces.)
```

Reply:
tweaks to @sentrivana 's suggestion